src/jit/ee_il_dll.cpp

```diff
@@ -398,13 +397,13 @@ unsigned CILJit::getMaxIntrinsicSIMDVectorLength(CORJIT_FLAGS cpuCompileFlags)
             return 32;
         }
     }
-#endif // !LEGACY_BACKEND
+#else  // !(defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND))
```

Comment:
Looks like this is wrong.
Should have been
```
#endif
#if defined(_TARGET_XARCH_) || defined(defined(_TARGET_ARM64_)
```

Reply:
This is not correct: defined(defined(_TARGET_ARM64_)